src/javascript-extract.js

```diff
@@ -0,0 +1,51 @@
+const acorn                      = require('acorn');
```

Comment:
ahem, not sure I've seen acorn pop up in the package.json, does it ? You should import it in the dependencies !

Reply:
@vperron, I introduced the `acorn` package during the last PR from me that you merged.

Reply:
acorn@^5.5.3 was already in the dependencies, AFAIK it is used to parse Javascript expressions in tag attributes :)